### PR TITLE
Display parent policies

### PR DESCRIPTION
**Pull request by macovedj**
Resolve https://github.com/opencollective/opencollective/issues/5326

# Description

Solves the expense aspect of [this](https://github.com/opencollective/opencollective/issues/5326)

# Screenshots

### CODE REVIEW

**Comment by macovedj** [comment]
I should follow up as there's a component called `ExpensiveInfoSidebar` that has a prop called `collective` that is currently being passed a project rather than a collective.  Ideally instead of just adding a ternary in the value passed, the prop name would be conditional as well so that it would be called a project when it was one.

**Comment by macovedj** [comment]
When I get a chance I'd like to add an e2e test.

File: components/expenses/ExpandableExpensePolicies.js [hunk 1]
```diff
@@ -15,12 +15,12 @@ const msg = defineMessages({
   },
 });
 
-const ExpandableExpensePolicies = ({ host, collective, ...props }) => {
+const ExpandableExpensePolicies = ({ host, organization, ...props }) => {
```
Comment (Betree) [inline components/expenses/ExpandableExpensePolicies.js:18]: I don't think we want to rename `collective` to `organization` here. For legacy reasons, in the vast majority of the codebase, "collective" is used as a generic word for all entities living in the `Collectives` table - regardless of their types. In new developments, we tend to use the word `Account` (which is the default in our public API).

File: components/expenses/ExpenseInfoSidebar.js [hunk 2]
```diff
@@ -83,7 +83,11 @@ const ExpenseInfoSidebar = ({ isLoading, host, collective, children }) => {
         </Container>
       </Box>
       {children && <Box my={50}>{children}</Box>}
-      <ExpandableExpensePolicies host={host} collective={collective} mt={50} />
+      <ExpandableExpensePolicies
+        host={host}
+        organization={collective?.type === 'PROJECT' ? collective.parent : collective}
```
Comment (Betree) [inline components/expenses/ExpenseInfoSidebar.js:88]: - Events have parents too, filtering on the type is too restrictive
- We only want to use the parent policies if the children (aka, project or event) does not define one

**Review by Betree — COMMENTED** [review]
Great job on adding tests for this one! We need a small followup on the feature & variable names, I'll look at that

File: pages/create-expense.js [hunk 3]
```diff
@@ -76,6 +76,7 @@ class CreateExpensePage extends React.Component {
       refetch: PropTypes.func,
       account: PropTypes.shape({
         id: PropTypes.string.isRequired,
+        parent: PropTypes.string,
```
Comment (Betree) [inline pages/create-expense.js:79]: Parent is an object (or shape), not a string

**Review by Betree — APPROVED** [review]

**Comment by Betree** [comment]
Added similar improvement in the contribution flow

image

**Pull request merged**